src/cdk/table/table.ts

```diff
@@ -79,6 +79,10 @@ import {
   getTableUnknownDataSourceError
 } from './table-errors';
 import {CDK_TABLE} from './tokens';
+import {
+  _TABLE_LAYOUT_STRATEGY, _TableLayout,
+  _TableLayoutStrategy
+} from '@angular/cdk/table/table-layout-strategy';
```

Comment:
I think you'll want to have an import path from `./table-layout-strategy`

Reply:
Done